stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if or (eq .Values.controller.kind "DaemonSet") (eq .Values.controller.kind "Both") }}
 {{- $useHostPort := .Values.controller.daemonset.useHostPort -}}
 {{- $hostPorts := .Values.controller.daemonset.hostPorts -}}
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
I think this could potentially be a breaking change for some users, so can we just make this configurable with the default being the existing API version?

Reply:
The **Deployment** resource is no longer available in `v1.16` under the `extensions/v1beta1 ` API. It is now only available in the `apps/v1` API. This was available in the `apps/v1` API since `v1.9`. This should be backward compatible as long as `.spec.selector` is explicitly defined within the **Deployment** resource.

Source: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/